### PR TITLE
chore(TU-590): Adopt ci-standard-checks

### PR DESCRIPTION
**Pull request by Thr44**

### CODE REVIEW

**Review by mathio — APPROVED** [review]

File: .github/workflows/ci-standard-checks.yml [hunk 1]
```diff
@@ -4,20 +4,23 @@ on:
...
+      - ready_for_review
     branches:
       - main
-
 jobs:
   ci-standard-checks:
-    runs-on: [ubuntu-latest]
-
+    runs-on:
+      - ubuntu-latest
```
Comment (mathio) [inline .github/workflows/ci-standard-checks.yml:18]: Should this run on `ubuntu-latest` or on [self-hosted automated-checks-ephemeral runner](https://github.com/Typeform/share-web/blob/main/.github/workflows/ci-standard-checks.yml#L19C35-L19C35)?

Comment (Thr44) [reply .github/workflows/ci-standard-checks.yml:18]: good question, it was running on ubuntu-latest previously, and the same for many other repos we have...

Comment (Thr44) [reply .github/workflows/ci-standard-checks.yml:18]: webflow is on ephemeral , so I will change this one to ephemeral too

File: .github/workflows/ci-standard-checks.yml [hunk 2]
```diff
@@ -4,23 +4,27 @@ on:
...
           fetch-depth: 0
-
       - name: CI Standard Checks
         uses: Typeform/ci-standard-checks@v1
         with:
           githubToken: ${{ secrets.GITHUB_TOKEN }}
-          dockerUsername: ${{ secrets.GITLEAKS_DOCKER_USERNAME }}
-          dockerPassword: ${{ secrets.GITLEAKS_DOCKER_PASSWORD }}
+          dockerUsername: ${{ secrets.DOCKER_USERNAME }}
+          dockerPassword: ${{ secrets.DOCKER_PASSWORD }}
```
Comment (Thr44) [inline .github/workflows/ci-standard-checks.yml]: on webflow we used these secrets

Comment (mumpo) [reply .github/workflows/ci-standard-checks.yml]: You can refer to this as the source of truth, so probably these `docker` variables are no longer necessary.

https://github.com/Typeform/.github/blob/main/workflow-templates/ci-standard-checks.yml

**Comment by Thr44** [comment]
badger ciStandardChecks

**Review by mumpo — APPROVED** [review]

File: .github/workflows/ci-standard-checks.yml (hunk 1 above)
Comment (mathio) [reply .github/workflows/ci-standard-checks.yml:18]: Just a note - we should not run on `ubuntu-latest` as those are more expensive than our runners. They are free for public repos only.

Comment (mumpo) [reply .github/workflows/ci-standard-checks.yml:18]: FYI this repo is public, so `ubuntu-latest` makes sense

**Pull request merged**